parm/archive/master_enkf.yaml.j2

```diff
@@ -74,6 +74,12 @@ datasets:
 
     {% endfor %}
 
+    {% set tmpl_dict = ({ '${ROTDIR}':ROTDIR,
+                          '${RUN}':RUN,
+                          '${YMD}':cycle_YMD,
+                          '${HH}':cycle_HH,
+                          '${MEMDIR}': "ensstat"}) %}
+    {% set COMIN_CONF = COM_CONF_TMPL | replace_tmpl(tmpl_dict) %}
```

Comment:
I think this should be moved to JGLOBAL_ENS_ARCHIVE_TARS by inserting a declaration here:

https://github.com/NOAA-EMC/global-workflow/blob/e131fb155c782159ac676b383d93fe0410e7f456/jobs/JGLOBAL_ENS_ARCHIVE_TARS#L9-L15

But I'm OK with this being handled as part of https://github.com/NOAA-EMC/global-workflow/issues/3767.